src/Core/Framework/Adapter/Twig/Extension/InstanceOfExtension.php

```diff
@@ -16,6 +16,10 @@ public function getTests(): array
         ];
     }
 
+    /**
+     * @param object $var
+     * @param class-string $class
+     */
     public function isInstanceOf($var, $class): bool
```

Comment:
why is that class not internal :see_no_evil: can we please use native types with the next major or now, if it is non breaking?